build/benchmarks/wawaka/interface-test.json

```diff
@@ -0,0 +1,10 @@
+[
```

Comment:
i though there was already an interface-test?

Reply:
This is the benchmark version of the test. It defines the number of iterations to run, for example. We use this test to establish a baseline for the marshaling costs of the json parameters/return values.